model_compression_toolkit/core/common/quantization/quantization_config.py

```diff
@@ -46,86 +46,45 @@ class QuantizationErrorMethod(Enum):
     HMSE = 6
 
 
+@dataclass
 class QuantizationConfig:
+    """
+    Class to wrap all different parameters the library quantize the input model according to.
+
+    Examples:
+        One may create a quantization configuration to quantize a model according to.
+        For example, to quantize a model's weights and activation using thresholds, such that
+        weights threshold selection is done using MSE, activation threshold selection is done using NOCLIPPING (min/max),
+        enabling relu_bound_to_power_of_2, weights_bias_correction,
+        one can instantiate a quantization configuration:
+
+        >>> import model_compression_toolkit as mct
+        >>> qc = mct.core.QuantizationConfig(activation_error_method=mct.core.QuantizationErrorMethod.NOCLIPPING, weights_error_method=mct.core.QuantizationErrorMethod.MSE, relu_bound_to_power_of_2=True, weights_bias_correction=True)
+
+
+        The QuantizationConfig instanse can then be passed to
+        :func:`~model_compression_toolkit.ptq.keras_post_training_quantization`
```

Comment:
specifically keras?

Reply:
Not specifically, this is just an example, I modified the text, let me know if you think that this is clear enough